DemoApp/DemoApp/Views/TableView.xaml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:modus="clr-namespace:Trimble.Modus.Components;assembly=Trimble.Modus.Components"
+             x:Class="DemoApp.TableView"
+             Title="DataGrid">
+  <ScrollView>
```

Comment:
Please remove the root ScrollView. That is not needed here

Reply:
ScrollView is needed since the table has more items now

Reply:
Hope this is addressed now

Reply:
It looks like the scrollview is still there. Like we discussed, remove the ScrollView and VeriticalStackLayout and have just a StackLayout

Reply:
ScrollView and VerticalStackLayout are removed and replaced with Grid